omnibus/config/projects/opscode-push-jobs-server.rb

```diff
@@ -35,6 +35,8 @@
 overrides_path = File.expand_path("../../../../omnibus_overrides.rb", __FILE__)
 instance_eval(IO.read(overrides_path), overrides_path)
 
+override :"chef-gem", :version => "14.1.12"
+override :ruby, :version => "2.4.4"
```

Comment:
tldr; I agree it would be reasonable to jump to 2.5.latest.

For the Chef Client we take the Ruby minor version upgrade only annually in our major release.

https://github.com/chef/chef-rfc/blob/6a58a6499bcc954a72191ec05a25eab4225b19b4/rfc092-dependency-update-cadence.md#ruby-cadence

Of course, with the client upgrading the ruby version brings in changes that can break people's cookbooks, which is why we have a specified cadence. I don't know that we have any similar concerns here since the user shouldn't have anything using this embedded ruby, or the chef-client for that matter. Which is why we're considering it safe to jump from Chef 12 to Chef 14 without a major release of these packages; we only consider that to affect our internal tools.